repos/fdbt-admin/src/App.tsx

```diff
@@ -54,12 +52,8 @@ const App = (): ReactElement => {
                     <Route path="/resendInvite/:username">
                         <ResendInvite isFullAdmin={isFullAdmin} />
                     </Route>
-                    {/* Exports */}
-                    <Route path="/listIncompleteExports">
-                        <ListIncompleteExports isFullAdmin={isFullAdmin} />
-                    </Route>
-                    <Route path="/deleteExport/:exportName">
-                        <DeleteExport isFullAdmin={isFullAdmin} />
+                    <Route path="/reporting">
+                        <Reporting isFullAdmin={isFullAdmin} />
                     </Route>
                     {/* Root */}
```

Comment:
I know this isn't part of your changes but do we need this comment as you removed the one above?

Reply:
We dont need it but I left it in because its not immediately obvious what the Route is